### PR TITLE
Closes #82 - Add saving password hash in database

### LINKED ISSUE #82: Add saving password hash  in database

### PR DESCRIPTION
**Pull request by AlexLviv07**

### CODE REVIEW

File: Timer.DAL/Timer.DAL/Repositories/UserRepository.cs [hunk 1]
```diff
@@ -172,5 +172,30 @@ public void Save()
         {
             timerContext.SaveChanges();
         }
+
+        public Task SetPasswordHashAsync(User user, string passwordHash)
```
Comment (roman-fay) [inline Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: wait a minute.. why do we need this methods again? where are they used right now?

Comment (roman-fay) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: just for the future use?

Comment (AlexLviv07) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: Yeah, you, right, we don`t explicitly use this three method, but it is used in createAsync method, is obligated inherite IUserPasswordStore interface

Comment (AlexLviv07) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: one minute, I make a screenshot error

Comment (AlexLviv07) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: image

Comment (roman-fay) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: btw, iUserPasswordStore implements IUserStore. It's unnecessary to have both of them on UserRepository
image

Comment (AlexLviv07) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: This error when I try add new user to database

Comment (AlexLviv07) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: Okey, then delete IUserStore

Comment (roman-fay) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: got it, just remove unnecessary IUserStore from Userrepository. Then we have 3 interfaces on that class which is not that bad (+ I don't want to hide IRepository under other interface). So at the end we can leave it like this:
public class UserRepository : IUserEmailStore<User, int>, IRepository<User>, IQueryableUserStore<User, int>, IUserPasswordStore<User, int>

without specifying additional interface

Comment (AlexLviv07) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: Ok, will change

Comment (AlexLviv07) [reply Timer.DAL/Timer.DAL/Repositories/UserRepository.cs]: Can I merge this code to develop branch?

**Pull request closed without merging**